Client.xcodeproj/project.pbxproj

```diff
@@ -2151,22 +2159,25 @@
 				D314E7F51A37B98700426A76 /* BrowserToolbar.swift */,
 				D3A994951A3686BD008AD1AC /* BrowserViewController.swift */,
 				D3BA7E0D1B0E934F00153782 /* ContextMenuHelper.swift */,
-				0BA1E02D1B046F1E007675AF /* ErrorPageHelper.swift */,
-				0BF42D361A7C0B8E00889E28 /* FaviconManager.swift */,
 				0BB5B30A1AC0AD1F0052877D /* LoginsHelper.swift */,
 				D3FA77831A43B2CE0010CD32 /* OpenSearch.swift */,
-				E47616C61AB74CA600E7DD25 /* ReaderModeBarView.swift */,
-				D31F95E81AC226CB005C9F3B /* ScreenshotHelper.swift */,
 				D308E4E31A5306F500842685 /* SearchEngines.swift */,
-				D34510871ACF415700EC27F0 /* SearchLoader.swift */,
 				D31A0FC61A65D6D000DC8C7E /* SearchSuggestClient.swift */,
-				59A68CCB63E2A565CB03F832 /* SearchViewController.swift */,
-				E68AEDAF1B18F81A00133D99 /* SwipeAnimator.swift */,
 				D3968F241A38FE8500CEFD3B /* TabManager.swift */,
 				D301AAED1A3A55B70078DD1D /* TabTrayController.swift */,
-				0BF0DBBF1A8598D50039F300 /* TransitionManager.swift */,
 				D3C744CC1A687D6C004CE85D /* URIFixup.swift */,
 				0BF0DB931A8545800039F300 /* URLBarView.swift */,
+				0BF42D361A7C0B8E00889E28 /* FaviconManager.swift */,
+				59A68CCB63E2A565CB03F832 /* SearchViewController.swift */,
+				0BF0DBBF1A8598D50039F300 /* TransitionManager.swift */,
+				E47616C61AB74CA600E7DD25 /* ReaderModeBarView.swift */,
+				D31F95E81AC226CB005C9F3B /* ScreenshotHelper.swift */,
+				D34510871ACF415700EC27F0 /* SearchLoader.swift */,
+				0BA1E02D1B046F1E007675AF /* ErrorPageHelper.swift */,
+				E68AEDAF1B18F81A00133D99 /* SwipeAnimator.swift */,
+				746D4E361B1E699C008F789F /* HashchangeHelper.swift */,
+				74C295261B21152000862FE3 /* AboutHomeHandler.swift */,
+				74203E241B276B3D007D481D /* SessionRestoreHandler.swift */,
```

Comment:
It's nice to keep these resources in alphabetical order. Can you right-click on the folder holding these, then choose "Sort By Type"?